dashboard/test/factories/pd_factories.rb

```diff
@@ -62,9 +63,9 @@
       evaluator.num_completed_surveys.times do
         enrollment = create :pd_enrollment, workshop: workshop
         if workshop.teachercon?
-          create :pd_teachercon_survey, pd_enrollment: enrollment
+          create :pd_teachercon_survey, pd_enrollment: enrollment, randomized_survey_answers: evaluator.randomized_survey_answers
         elsif workshop.local_summer?
-          create :pd_local_summer_workshop_survey, pd_enrollment: enrollment
+          create :pd_local_summer_workshop_survey, pd_enrollment: enrollment, randomized_survey_answers: evaluator.randomized_survey_answers
```

Comment:
am I misreading things, or does the `pd_local_summer_workshop_survey` factory actually expect the evaluator to be passed in its entirety, rather than just the `randomized_survey_answers` option?